templates/search/items_list.html

```diff
@@ -15,7 +15,7 @@
     </td>
     <td class="table-td table-name">
       <a class="table-link" href="{{ item.get_absolute_url }}">
-        {{ item.name }} {{ item.mark }} {{ item.specification }}
```

Comment:
You missed `specification` here

Reply:
@artemiy312 , now item has no specification field.
Series feature will hadle specification field content in more correct way